docs/topics/db/models.txt

```diff
@@ -1417,6 +1417,79 @@ different database tables).
 Django will raise a :exc:`~django.core.exceptions.FieldError` if you override
 any model field in any ancestor model.
 
+
+.. _proxy-models-permissions-topic:
+
+Permissions
+-----------
+
+Proxy models work exactly the same way as concrete models and permissions are
+automatically created using their own content type. Also, proxy models don't
+inherit the permissions of the concrete model they subclass::
+
+    class Person(models.Model):
+        class Meta:
+            permissions = (('can_eat_pizzas', 'Can eat pizzas'),)
+
+    class Student(Person):
+        class Meta:
+            proxy = True
+            permissions = (('can_deliver_pizzas', 'Can deliver pizzas'),)
+
+    >>> # Fetch the content type for the proxy model.
+    >>> content_type = ContentType.objects.get_for_model(
+    ...     Student,
+    ...     for_concrete_model=False,
+    ... )
+    >>> student_permissions = Permission.objects.filter(content_type=content_type)
+    >>> [p.codename for p in student_permissions]
+    ['add_student', 'change_student', 'delete_student', 'view_student',
+    'can_deliver_pizzas']
+
+    >>> for permission in student_permissions:
+    ...     user.user_permissions.add(permission)
+
+    >>> user.has_perm('foobar.add_person')
+    False
+    >>> user.has_perm('foobar.can_eat_pizzas')
+    False
+    >>> user.has_perms('foobar.add_student', 'foobar.can_deliver_pizzas')
+    True
+
+ContentType
+-----------
+
+Proxy models have their own content type which are used to create permissions
```

Comment:
This discussion seems unrelated to the fix at hand.

Reply:
This is kind-of half-ref to [#22270](https://code.djangoproject.com/ticket/22270): permissions go via content types... If it's a worthwhile clarification it could go in by itself (in another PR).

Reply:
@carltongibson Do you want me to move this piece to #10454 ? Or in a new PR altogether?